### ISSUE TITLE
Choose "no" on "auto-start" print for Octoprint always start the print

### ISSUE THREAD

**Issue by gege2b** [alexrj/Slic3r#3655]
hi all and happy new year :)

### Version
_Version of Slic3r used goes here_
Current dev ( 586d95b28a8e473bdcbd959f1e37d215ae531df5 )

### Operating system type + version

Kubuntu 16.60 x64

### Behavior
* _Describe the problem_
Sending Gcode to Octoprint, when prompted if the print should start as soon as the file is uploaded, I choose "no" but the print start anyway. It also start if I choose yes (as expected).

So basically, the print always start, no matter what I choose when prompted

* _Steps needed to reproduce the problem_

Load a model, slice it, send to printer, click 'no' for auto-start the print, look the print begin

Another question related to this new octoprint stuff : is there a way to disable this (prompting for the filename AND for autostart) ?

**Comment by mloebl** [comment]
Same problem here too on Linux.  

I also wouldn't mind an option to disable that prompt to ask about auto starting.

**Comment by gege2b** [comment]
OK at least, I'm not the only one who got this issue :)
For the option, I suggested something in #3657

**Comment by mloebl** [comment]
@gege2b Nope :)  Meant to log this last week when I hit the issue, but forgot... remembered today, and saw you beat me to it.

**Comment by gege2b** [comment]
aha :)

after digging a bit into the code, I think I got the culprit
Plater.pm line 1402 : 
 ```            print => $self->{send_gcode_file} ? 1 : 0,
should be : 
```            print => $self->{send_gcode_file_print} ? 1 : 0,

Still not tested, my octoprint server is not reachable right now

**Comment by gege2b** [comment]
OK I can confirm (tested right now)
@alexrj or @lordofhyphens do you want me to do a PR for this tiny fix ?

**Comment by lordofhyphens** [comment]
Go ahead.
